geode-core/src/main/java/org/apache/geode/internal/cache/control/InternalResourceManager.java

```diff
@@ -288,20 +294,20 @@ public RebalanceFactory createRebalanceFactory() {
 
   @Override
   public Set<RebalanceOperation> getRebalanceOperations() {
-    synchronized (this.inProgressOperationsLock) {
-      return new HashSet<RebalanceOperation>(this.inProgressOperations);
+    synchronized (this.inProgressRebalanceOperationsLock) {
+      return new HashSet<>(this.inProgressRebalanceOperations);
```

Comment:
Maybe an unmodifiable set wrapped around a concurrent map?

Also the use of `this.` qualifier is not necessary in this context.

Reply:
Done